### ISSUE TITLE
Crazy amounts of stutter

### ISSUE THREAD

**Issue by ShadicX** [Kaldaien/FAR#101]
As the title says i've been getting crazy amounts of stutter anywhere in the game, i have the latest version (0.7.0.14) been messing with the settings such as lowering global illumination to low, changing the frame rate limiter to no avail.
Any ideas on how to fix this.

My specs if it helps
Nvidia gtx 1070
 i7-6700HQ

**Comment by Idearum** [comment]
Does the issue still occur even without FAR enabled? (rename dinput8.dll to something else)

**Comment by ShadicX** [comment]
Yep stutters without FAR enabled, should i delete FAR and then download it again?

**Comment by Idearum** [comment]
If the game stutters even without FAR enabled then the issue is something else entirely. I recommend you to widen your troubleshooting and attempt to find what's causing the issue to begin with (perhaps you've it installed on a HDD and not a SSD?). After you've finally managed to solve it you can go ahead and install FAR again.
